aip/general/0213.md

```diff
@@ -120,7 +120,7 @@ and the definitive list is always [the code][type], several types deserve note:
 [time_of_day]: https://github.com/googleapis/googleapis/blob/master/google/type/timeofday.proto
 <!-- prettier-ignore-end -->
 
-## Appendix: Adding to common protos
+## Adding to common protos
```

Comment:
Should this be `###`? It is done elsewhere in this PR

Reply:
I think you're right... I was think it's a little outside of guidance, but AIP-8 only gives `###` subsections.

Will fix.